vcr/api/oidc4vci_v0/wallet.go

```diff
@@ -3,16 +3,25 @@ package oidc4vci_v0
 import (
 	"context"
 	"encoding/json"
+	"github.com/nuts-foundation/go-did/did"
 	"github.com/nuts-foundation/nuts-node/core"
 	"github.com/nuts-foundation/nuts-node/vcr/api/oidc4vci_v0/types"
 	"net/url"
 )
 
 func (w Wrapper) GetOAuth2ClientMetadata(_ context.Context, request GetOAuth2ClientMetadataRequestObject) (GetOAuth2ClientMetadataResponseObject, error) {
-	return GetOAuth2ClientMetadata200JSONResponse(w.HolderRegistry.GetWallet(request.Did).Metadata()), nil
+	id, err := did.ParseDID(request.Did)
+	if err != nil {
+		return nil, core.InvalidInputError("invalid did")
```

Comment:
Probably just a 404 Not Found? (since the caller can't do anything with "invalid DID")